.ci/jenkins_job_build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Need the repo to know which tests to run.
+cc_repo=$1
+
+tests_repo="github.com/clearcontainers/tests"
+mkdir -p ${HOME}/go
+
+# Export all environment variables needed.
+export GOROOT="/usr/local/go"
+export GOPATH=${HOME}/go
+export PATH=${GOPATH}/bin:/usr/local/go/bin:/usr/sbin:${PATH}
+export CI=true
+
+# Download and build goveralls binary in case we need to submit the code
+# coverage.
+if [ ${COVERALLS_REPO_TOKEN} ]
+then
+	go get github.com/mattn/goveralls
+fi
+
+# Get the repository and move HEAD to the appropriate commit.
+go get ${cc_repo} || true
```

Comment:
you might want `go get -d` here and to drop the `|| true` ?

Reply:
`go get -d` never worked, that's why our script are all using `|| true`.

Reply:
hmm, do we know if there is a reason why - it would be nice to fix that. We don't have to block due to it though. /cc @jodh-intel for any thoughts/input

Reply:
Huh? `go get -d` never worked? How can that not work?